### PR TITLE
Added support for building multi-arch Docker images for elasticmq-native

### PR DESCRIPTION
**Pull request by rojekp**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -283,6 +287,14 @@ lazy val nativeServer: Project = (project in file("native-server"))
     dockerEntrypoint := Seq("/sbin/tini", "--", "/opt/docker/bin/elasticmq-native-server", "-Dconfig.file=/opt/elasticmq.conf"),
     dockerUpdateLatest := true,
     dockerExposedPorts := Seq(9324,9325),
+    dockerBuildOptions := dockerBuildOptions.value :+ "--platform=linux/arm64,linux/amd64" :+ "--push",
+    dockerBuildCommand := {
```
Comment (adamw) [inline build.sbt:291]: maybe this can be part of `dockerBuildxSettings` and hence not repeated?

File: build.sbt [hunk 2]
```diff
@@ -349,7 +364,7 @@ lazy val uiSettings = Seq(
 lazy val dockerBuildxSettings = Seq(
   createBuildx := {
     streams.value.log("Creating docker buildx instance")
-    haltOnCmdResultError(Process("docker buildx create --use --name multi-arch-builder", baseDirectory.value).!)
+    Process("docker buildx create --use --name multi-arch-builder", baseDirectory.value).!
```
Comment (adamw) [inline build.sbt:367]: we don't want to halt? :)

File: build.sbt (hunk 1 above)
Comment (rojekp) [reply build.sbt]: Moved that out like suggested.

File: build.sbt (hunk 2 above)
Comment (rojekp) [reply build.sbt:351]: Well, that command will return either success with new Buildx instance or just info about existing instance.

Comment (adamw) [reply build.sbt:351]: So if the instance exist, the command ends in error?

**Pull request merged**